.circleci/config.yml

```diff
@@ -98,6 +98,32 @@ jobs:
           key: build-cache-{{ .Revision }}
           paths:
             - dist
+      - store_artifacts:
+          path: dist/mascara
+          destination: builds/mascara
+      - store_artifacts:
+          path: builds
+          destination: builds
+      - run:
+          name: build:announce
+          command: |
```

Comment:
this just seems like it should go in a script?

Reply:
that sounds reasonable